HW12/hw_12_task_2.py

```diff
@@ -0,0 +1,13 @@
+# 2. (необов'язкове виконання) Написати власний менеджер контексту, задачею якого буде друкувати "==========" – 10 знаків
+# дорівнює перед виконанням коду та після виконання коду, таким чином виділяючи блок коду символами дорівнює.
+
+class MyContextManager:
+    def __enter__(self):
+        print("=" * 10)
+        return 1
```

Comment:
nit: there is no reason to `return 1`. it can be deleted. Probably I didn't get the idea)